src/hotspot/share/opto/loopnode.cpp

```diff
@@ -4863,29 +4862,16 @@ void PhaseIdealLoop::build_and_optimize() {
      C->set_major_progress();
   }
 
-  // Convert scalar to superword operations at the end of all loop opts.
+  // Auto-vectorize main-loop
   if (C->do_superword() && C->has_loops() && !C->major_progress()) {
```

Comment:
Should we rename `do_superword()` too?

Reply:
@vnkozlov yes, but later, I would say. It simply caries the same values as `UseSuperWord`. I guess eventually we may have different AutoVectorization components, and it may be better to have a unified AutoVectorization option. Is it ok to leave that to the future, and leave it in for now?

Reply:
Yes, we can do it for later.